### PR TITLE
Add Structure Spacing Mappings

### PR DESCRIPTION
**Pull request by TheBrokenRail**
I don't know why Mojang would move this from the previously sensible location of methods inside StructureFeature, but here it is.

### CODE REVIEW

File: mappings/net/minecraft/world/gen/feature/StructureSpacing.mapping [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_5299 net/minecraft/world/gen/feature/StructureSpacing
+	FIELD field_24647 STRUCTURE_SPACING Lcom/google/common/collect/ImmutableMap;
+	CLASS class_5300 StructureSpacingInformation
```
Comment (YanisBft) [inline mappings/net/minecraft/world/gen/feature/StructureSpacing.mapping]: Just `Information` should suffice as it's an inner class

File: mappings/net/minecraft/world/gen/feature/StructureSpacing.mapping [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+		FIELD field_24648 CODEC Lcom/mojang/serialization/Codec;
+		FIELD field_24649 spacing I
+		FIELD field_24650 seperation I
```
Comment (YanisBft) [inline mappings/net/minecraft/world/gen/feature/StructureSpacing.mapping]: `separation`

File: mappings/net/minecraft/world/gen/feature/StructureSpacing.mapping (hunk 1 above)
Comment (TheBrokenRail) [reply mappings/net/minecraft/world/gen/feature/StructureSpacing.mapping]: Fixed!

File: mappings/net/minecraft/world/gen/feature/StructureSpacing.mapping (hunk 2 above)
Comment (TheBrokenRail) [reply mappings/net/minecraft/world/gen/feature/StructureSpacing.mapping]: Fixed!

File: mappings/net/minecraft/world/gen/feature/StructureSpacingInformation.mapping [hunk 3]
```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_5314 net/minecraft/world/gen/feature/StructureSpacingInformation
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/StructureSpacingInformation.mapping]: Is the field called `information` in the codec for structure spacing config?

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/world/gen/feature/StructureSpacingInformation.mapping (hunk 3 above)
Comment (TheBrokenRail) [reply mappings/net/minecraft/world/gen/feature/StructureSpacingInformation.mapping]: It doesn't have a clear name in the Codec as far as I can tell? I think ```Information``` just makes sense as a suffix.

File: mappings/net/minecraft/world/gen/feature/StructureSpacingConfig.mapping [hunk 4]
```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_5311 net/minecraft/world/gen/feature/StructureSpacingConfig
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/StructureSpacingConfig.mapping]: I personally favor name from https://github.com/FabricMC/yarn/pull/1384/files#diff-e032b9ef76cdc917941321473aec9056R1 better.

File: mappings/net/minecraft/world/gen/feature/StructureSeparationDataFix.mapping [hunk 5]
```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_5299 net/minecraft/world/gen/feature/StructureSeparationDataFix
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/StructureSeparationDataFix.mapping]: data fixers shall go into `net/minecraft/datafixer/fix` package

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/world/gen/feature/StructureSeparationDataFix.mapping (hunk 5 above)
Comment (TheBrokenRail) [reply mappings/net/minecraft/world/gen/feature/StructureSeparationDataFix.mapping]: Fixed!

File: mappings/net/minecraft/world/gen/feature/StructureSpacingConfig.mapping (hunk 4 above)
Comment (TheBrokenRail) [reply mappings/net/minecraft/world/gen/feature/StructureSpacingConfig.mapping]: What do you think of the new names I put in the PR?

**Comment by liach** [comment]
Mind merge the current 1.16 pre2 stuff and remove duplicate mappings?

**Comment by TheBrokenRail** [comment]
Done!

**Review by YanisBft — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Pull request merged**